### ISSUE TITLE
FastReport.Utils.CompilerException: Error CS0006 Microsoft.CSharp cannot be found

### ISSUE THREAD

**Issue by inhomecoder** [FastReports/FastReport#436]
Hi, the same error, that has already been reported:
Prerequisites: FastReport v. 2022.2.9.0
WinForms 4.8

CS-Code:

  Report report = new Report(); 
            report.Load(@"C:\Title.frx");
            DataSet data = new DataSet();
            data.ReadXml(@"C:\DataSource.xml"); 
            report.RegisterData(data.Tables[0], "MySource");
            report.Prepare(); //FastReport.Utils.CompilerException: Error CS0006

frx file in attachment

Can anybody tell me, what's wrong here?

[Title.zip](https://github.com/FastReports/FastReport/files/8739130/Title.zip)

**Comment by larsmaguina** [comment]
I use the library:
FastReport.OpenSource.Data.MsSql - 2021.4.0

I solved it putting to the same version:
FastReport.OpenSource
FastReport.OpenSource.Export.PdfSimple

**Comment by MaratAlaev** [comment]
Hello!

Is the package installed? https://www.nuget.org/packages/FastReport.Compat/

Best regards 
Marat

**Comment by SMX-Prog** [comment]
I found the solution in another post 
I add some more code to help.

It helped me solve my problem with the 2025 version.
As soon as I added a table inside the report (even without filling the cells) the problem occured. 

These 3 lines of code can each solve the problem. I tested them all one by one.

var _ = typeof(Microsoft.CSharp.RuntimeBinder.Binder).Assembly; //Best way to solve the problem (said Chat GPT)

typeof(Microsoft.CSharp.RuntimeBinder.RuntimeBinderException).GetType();    

AppDomain.CurrentDomain.Load("Microsoft.CSharp"); 

They all work!
